src/utils.h

```diff
@@ -157,23 +157,23 @@ namespace pyne {
     ~FileNotFound () throw () {};
 
     /// constructor with the filename \a fname.
-    FileNotFound(std::string fname)
+    FileNotFound(std::string fname) : FNF_message("File not found: ")
     {
       filename = fname;
     };
 
     /// Creates a helpful error message.
     virtual const char* what() const throw()
     {
-      std::string FNFstr ("File not found: ");
       if (!filename.empty())
-        FNFstr += filename;
+        FNF_message += filename;
 
-      return (const char *) FNFstr.c_str();
+      return (const char *) FNF_message.c_str();
```

Comment:
`c_str` returns a `const char *` by default, so the c-style cast here can go

Reply:
cleaned this up